src/main/python/main/ayab/image.py

```
@@ -111,12 +114,27 @@ def __open(self, filename: str) -> None:
             self.image = Image.open(filename)
+        if suffix == ".png":
+            # check metadata for memo information
+            self.image.load()
+            if "Comment" in self.image.info and len(str(self.image.info["Comment"])) > 0:
+                comment = str(self.image.info["Comment"])
+                if comment.startswith("AYAB:"):
+                    # update memo information
+                    for i in range(len(comment) - 5):
+                        try:
+                            self.memos.append(comment[i + 5])
+                        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)